chsdi/tests/functional/test_helpers.py

```diff
@@ -68,6 +69,16 @@ def test_check_url(self):
         url = 'http://admin.ch'
         self.assertEqual(url, check_url(url, config))
 
+    def test_sanitize_url(self):
+        url = 'http://www.cwi.nl:80/%7Eguido/Python.html'
+        result = sanitize_url(url)
+        self.assertEqual(result, result)
```

Comment:
This test can never fail. (we compare 2 same objects). What's the goal of this test?